_schemaV2.graphql

```diff
@@ -11842,6 +11842,25 @@ type ImageURLs {
   normalized: String
 }
 
+# Schema for everything infinite discovery-related
+type InfiniteDiscovery {
```

Comment:
again, not strongly opinionated, but I intentionally introduced a new field, because:

- it returns a connection and I think it should be explicit in the name
- `discoverArtworks` seemed too generic to me, but @joeyAghion suggests to not use a name which is too project-specific.